constants/src/lib.rs

```diff
@@ -0,0 +1,51 @@
+use convert_case::Casing;
+use utils::constants::PRODUCT_DOMAIN_NAME;
+pub use utils::PRODUCT_NAME;
+
+/// Name of the product.
+pub fn product_pascal() -> String {
+    PRODUCT_NAME.to_case(convert_case::Case::Pascal)
+}
+
+/// Helm release name label's key.
+pub fn helm_release_name_key() -> String {
+    format!("{PRODUCT_DOMAIN_NAME}/release")
+}
+
+/// Upgrade job container image name.
+pub fn upgrade_job_img() -> String {
+    format!("{PRODUCT_NAME}-upgrade-job")
+}
+/// Upgrade job container image name.
+pub fn upgrade_job_container_name() -> String {
+    format!("{PRODUCT_NAME}-upgrade-job")
+}
+/// UPGRADE_EVENT_REASON is the reason field in upgrade job.
+pub fn upgrade_event_reason() -> String {
+    format!("{}Upgrade", product_pascal())
```

Comment:
Probably needs a space before Upgrade?

Reply:
No, that's how it was "MayastorUpgrade"
> pub(crate) const UPGRADE_EVENT_REASON: &str = "MayastorUpgrade";

Reply:
Ahh okay.

Reply:
I believe the label is still `openebs.io/logging` for the loki scrape targets. We don't need to touch the scrape configs.

Reply:
That will have to be done manually, in fact in a bunch of places such as docs.yaml, chart,yaml, product.yaml and values.yaml in a few places.